src/utils/test.ts

```diff
@@ -0,0 +1,9 @@
+import { expect } from "chai";
+import getHyperLinkFromPath from "./getHyperLinkFromPath";
+
+describe("getHyperLinkFromPath Function", () => {
```

Comment:
You need more test cases to handle EDGE cases

EX:
src
./src
./src/
./src////
/.wwqe
.//src/
Always good to capture majority of test cases

Reply:
so 
./src
./src/
./src////
/.wwqe
.//src/
obviously are wrong because github API wouldn't generate them... but I haven't predict anything inside the function to check if the input is correct or throws an error... should I provide such thing?

Reply:
Yes your function should always handle edge cases such as bad paths that aren't supported (Throw an Error)